android/src/main/java/com/reactnativecommunity/webview/RNCWebView.java

```diff
@@ -0,0 +1,36 @@
+package com.reactnativecommunity.webview;
+
+import android.widget.FrameLayout;
+
+import com.facebook.react.uimanager.ThemedReactContext;
+
+public class RNCWebView extends FrameLayout {
```

Comment:
We've wrapped the old RNCWebView (now, `InternalWebView`) inside a FrameLayout. This allows use to re-attach the view as needed.